pkg/http/server.go

```diff
@@ -19,8 +20,8 @@ func (*Server) NeedLeaderElection() bool {
 	return false
 }
 
-func NewServer(address string, client client.Client) *Server {
-	return &Server{&http.Server{Addr: address}, client}
+func NewServer(address, defaultNamespace string, client client.Client) *Server {
```

Comment:
Can we make the param ordering consistent between `NewServer` and `http.Server`?